### ISSUE TITLE
Plugin "RPCDump" in rpcdump.py is not a subclass of either PortScan, ServiceScan, or Report.

### ISSUE THREAD

**Issue by somerandomdudeonetheinternet** [Tib3rius/AutoRecon#159]
Multiple errors getting thrown since the latest kali updates. Will this affect functionality?
```
┌──(root💀kali)-[~/<...REDACTED...>]
└─# autorecon thomaswreath.thm                                                                                                     1 ⨯
Plugin "RPCDump" in rpcdump.py is not a subclass of either PortScan, ServiceScan, or Report.
Plugin "MiniImpacketShell" in smbclient.py is not a subclass of either PortScan, ServiceScan, or Report.
Plugin "SMBConnection" in smbclient.py is not a subclass of either PortScan, ServiceScan, or Report.
[*] Scanning target thomaswreath.thm
```

**Comment by Tib3rius** [comment]
@somerandomdudeonetheinternet is this the latest version of AutoRecon? This should have been fixed in v2.0.20. You can determine the version of AutoRecon using `autorecon --version`

**Comment by somerandomdudeonetheinternet** [comment]
> @somerandomdudeonetheinternet is this the latest version of AutoRecon? This should have been fixed in v2.0.20. You can determine the version of AutoRecon using `autorecon --version`

It was at the time of the issue creation
